CalibMuon/DTCalibration/plugins/DTTTrigCalibration.h

```diff
@@ -88,5 +90,6 @@ class DTTTrigCalibration : public edm::EDAnalyzer {
   std::unique_ptr<DTTimeBoxFitter> theFitter;
   // The module for t0 subtraction
   std::unique_ptr<DTTTrigBaseSync> theSync;  //FIXME: should be const
```

Comment:
Maybe time to make it `const` as the comment suggests?

Reply:
Neither `std::unique_ptr<DTTTrigBaseSync const> theSync;` nor `const std::unique_ptr<DTTTrigBaseSync> theSync;` compile. So it is unclear what that legacy comment wants.